src/services/chains.rs

```diff
@@ -0,0 +1,36 @@
+use crate::cache::cache_operations::RequestCached;
+use crate::config::{
+    base_config_service_url, chain_info_cache_duration, chain_info_request_timeout,
+};
+use crate::models::chains::ChainInfo;
+use crate::models::commons::Page;
+use crate::utils::context::Context;
+use crate::utils::errors::ApiResult;
+
+pub async fn get_chains_paginated(
+    context: &Context<'_>,
+    limit: Option<&String>,
```

Comment:
In all our current views we do not expose `limit`and `offset`. 

While in this case it would not really pose an issue I like that the clients don't make any assumptions on how the pagination works. They should just assume that there is a "cursor" in the url ... and even that they should probably never know as they should just use the returned `next` url. See https://github.com/gnosis/safe-client-gateway/blob/main/src/routes/transactions.rs#L112

Reply:
I agree. I will remove this. This can be easily adjusted if required 👍

Reply:
This was added back because `limit` is being used by the clients